server.go

```diff
@@ -285,7 +295,7 @@ func (server *Server) signDropperRoute(w http.ResponseWriter, r *http.Request, s
 	authorizationToken := authorizationContext.AuthorizationToken
 
 	queries := r.URL.Query()
-	isMetatxDrop := queries.Has("is_metatx_drop")
+	noMetatx := queries.Has("no_metatx")
```

Comment:
I really don't like these query parameters that have no values.

Can you make this part of the JSON request body?

Reply:
Done